<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabMainWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_Match">
       <attribute name="title">
        <string>Match</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_Match">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_t1">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_t1_name">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_t1_name">
                <property name="text">
                 <string>Team 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_t1_name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_team1_sn">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_t1_sn">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_t1_sn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Short name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_t1_sn"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_t1_rank">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_t1_rank">
                  <item>
                   <widget class="QLabel" name="label_t1_rank">
                    <property name="minimumSize">
                     <size>
                      <width>27</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Rank </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_t1_rank">
                    <property name="minimumSize">
                     <size>
                      <width>72</width>
                      <height>0</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>None</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_side">
                  <item>
                   <widget class="QLabel" name="label_side">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="baseSize">
                     <size>
                      <width>0</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Side</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_side_attack">
                    <property name="baseSize">
                     <size>
                      <width>0</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>A</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_side_defence">
                    <property name="baseSize">
                     <size>
                      <width>0</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>D</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_general">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_button">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_swap">
                <property name="text">
                 <string>Swap</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_update">
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_reset">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_map">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_map">
                <property name="text">
                 <string>Map</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_map">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_score">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label_score">
                <property name="text">
                 <string>Score</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_score">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QSpinBox" name="spinBox_t1_score"/>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_t2_score">
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_team2">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_team2_name">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_t2_name">
                <property name="text">
                 <string>Team 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_t2_name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_t2_2">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_t2_rank">
                  <item>
                   <widget class="QComboBox" name="comboBox_t2_rank">
                    <property name="minimumSize">
                     <size>
                      <width>72</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::RightToLeft</enum>
                    </property>
                    <item>
                     <property name="text">
                      <string>None</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_t2_rank">
                    <property name="minimumSize">
                     <size>
                      <width>27</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Rank</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Minimum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>10</width>
                    <height>22</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_t2_shortName">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_t2_sn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>Short name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_t2_sn"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Cast">
       <attribute name="title">
        <string>Cast</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
